#!/usr/bin/env python
PACKAGE = "coman_control"

import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("publish_frequency", double_t, 0, "Rate of published desired joint values", 1000, 10, 1000)
gen.add("execute_trajectory", bool_t, 0, "Start execution of the trajectory", False)
gen.add("reset_joints", bool_t, 0, "Set all joints values to 0", False)
gen.add("reset_trajectory", bool_t, 0, "Set execution of the trajectory from beginning", False)

exit(gen.generate(PACKAGE, "send_trajectories", "send_trajectories"))
